File: src/main/java/com/kisman/cc/mixin/mixins/accessor/INetworkManager.java
package com.kisman.cc.mixin.mixins.accessor;

import net.minecraft.network.Packet;

public interface INetworkManager {

    Packet<?> sendPacketNoEvent(Packet<?> packetIn);

    Packet<?> sendPacketNoEvent(Packet<?> packetIn, boolean post);

}

File: src/main/java/com/kisman/cc/mixin/accessors/INetworkManager.kt
package com.kisman.cc.mixin.accessors

import io.netty.channel.ChannelHandlerContext
import net.minecraft.network.Packet

/**
 * @author _kisman_
 * @since 21:58 of 10.01.2023
 */
interface INetworkManager {
    fun sendPacketNoEvent(
        packet : Packet<*>?,
        post : Boolean
    )

    fun sendPacketNoEvent(
        packet : Packet<*>?
    )

    fun channelRead00(
        channel : ChannelHandlerContext?,
        packet : Packet<*>
    )
}